### PR TITLE
Change competition terms and conditions hash, change CHF asset mock a…

### PR DESCRIPTION
**Pull request by GauthamN**
…ddress

Please describe the purpose of this pull request below.
The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)

Copy/paste from CHANGELOG.md is good

### Added
- NoComplianceCompetition contract
- Deployment settings for competition-replica on kovan

### Changed
- T and C hash in competition contract
- T and C hash to bytes instead of bytes32

### Fixed
- Rounding error in tests

### CODE REVIEW

File: utils/deploy/contracts.js [hunk 1]
```diff
@@ -138,20 +138,25 @@ async function deployEnvironment(environment) {
...
-        ethTokenAddress, deployed.CanonicalPriceFeed.address, deployed.OnlyManagerCompetition.address
+        ethTokenAddress, deployed.CanonicalPriceFeed.address, complianceAddress
       ],
       () => {}, true
     );
     deployed.FundRanking = await deployContract("FundRanking", opts);
     const blockchainTime = await getChainTime();
-    deployed.Competition = await deployContract("competitions/Competition", opts, [mlnAddr, chfAddress, deployed.Version.address, accounts[0], blockchainTime, blockchainTime + 864000, 22 * 10 ** 18, 10 ** 24, 1000]);
-    await deployed.Competition.instance.batchAddToWhitelist.postTransaction(opts, [10 ** 25, [accounts[0]]]);
+    deployed.Competition = await deployContract("competitions/Competition", opts, [mlnAddr, chfAddress, deployed.Version.address, accounts[0], blockchainTime, blockchainTime + 864000, 20 * 10 ** 18, 10 ** 24, 1000]);
```
Comment (jennazenk) [inline utils/deploy/contracts.js]: could we add after this a transfer of MLN tokens to the Competition contract ?

File: src/compliance/NoComplianceCompetition.sol [hunk 2]
```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.19;
+
+import "./NoCompliance.sol";
+
+/// @title Compliance Contract
+/// @author Melonport AG <team@melonport.com>
+/// @notice Only allow manager to invest in the Fund.
```
Comment (jennazenk) [inline src/compliance/NoComplianceCompetition.sol]: outdated comment, would be "Anyone is allowed to invest in the Fund". 

What is the difference between NoComplianceCompetition and NoCompliance ?

Comment (GauthamN) [reply src/compliance/NoComplianceCompetition.sol]: Changed the comment. It's basically the same with the added isCompetitionAllowed function that always returns true (actual version checks for whitelist and competition is active conditions)

File: utils/deploy/contracts.js (hunk 1 above)
Comment (GauthamN) [reply utils/deploy/contracts.js]: Yeah but could we assume the deployer has MLN tokens?

Comment (jennazenk) [reply utils/deploy/contracts.js]: think we can but better to check with @travs

File: src/compliance/NoComplianceCompetition.sol (hunk 2 above)
Comment (jennazenk) [reply src/compliance/NoComplianceCompetition.sol]: gotcha

**Pull request merged**